src/streamlink_cli/argparser.py

```diff
@@ -813,11 +813,20 @@ def build_parser():
         "--hls-segment-key-uri",
         metavar="URI",
         type=str,
-        help="""
+        help=r"""
```

Comment:
Please don't use a raw string here. Escape the `{{variables}}` instead, just like it's done everywhere else.

Reply:
Will do!